content/docs/howto/negate-a-disjunction/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Negate a disjunction
```

Comment:
You've taken the title from the issue I created (https://github.com/cue-lang/docs-and-content/issues/160), but I'm going to ask @myitcv to confirm that "negate" is the term we want to surface here.

Whatever verb we end up with, please align with these forms across the URL path and page title:

- URL path: /docs/howto/negate-a-disjunction (notice that it reads naturally)
- Page title: Negating a disjunction (this enables it to be formatted on other pages as `Howto Guide: Negating a disjunction`, without requiring the "howto" element to be an integral part of the meaning of the title)

Reply:
I'll await his answer; *complement* (set theory) and *inverse* spring to mind as well.

Reply:
I'll push forward and submit this as-is; we can always update the title later if needed :-)